wandb/sdk/artifacts/artifact.py

```diff
@@ -537,6 +537,25 @@ def description(self) -> str | None:
         """A description of the artifact."""
         return self._description
 
+    @property
+    def url(self):
+        """
+        Constructs the URL of the artifact.
+
+        Returns:
+            str: The URL of the artifact.
+        """
+        if self.collection.is_sequence():
+            return f"{self._client.app_url}/{self._entity}/{self._project}/artifacts/{self._type}/{self._name}"
+        else:
+            if self._project.startswith("wandb-registry-"):
+                org = InternalApi()._fetch_orgs_and_org_entities_from_entity(self._entity)[0]
+                return f"{self._client.app_url}orgs/{org.display_name}/registry/{self._type}?selectionPath={self._entity}%2F{self._project}%2F{self._name.split(':')[0]}&view=membership&version={self._version}"
```

Comment:
`selectionPath={self._entity}` can also be `selectionPath={org.entity_name}`, not sure if there's any preference?